CONTRIBUTING.md

```diff
@@ -32,33 +20,70 @@ The following is a set of guidelines for contributing to Redash. These are guide
 - [Release Method](#release-method)
 - [Code of Conduct](#code-of-conduct)
 
+## Quick Links:
+
+- [Feature Requests](https://discuss.redash.io/c/feature-requests)
+- [Documentation](https://redash.io/help/)
+- [Blog](https://blog.redash.io/)
+- [Twitter](https://twitter.com/getredash)
+
+---
 ## How can I contribute?
 
-### Reporting Bugs
+## Reporting Bugs
```

Comment:
Why reduce the level? It's a sub headline after "How can I contribute?". No?

Reply:
I reverted this change.